pkg/controllers/streaming/processor_controller.go

```diff
@@ -682,6 +684,21 @@ func (r *ProcessorReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			}),
 		}
 	}
+	enqueueNamedTrackedConfigMap := &handler.EnqueueRequestsFromMapFunc{
```

Comment:
Let's combine this with enqueueTrackedResource similar to what @joshrider came up with:
https://github.com/projectriff/system/blob/28aac56da45362d4852822be5f93c2825a539533/pkg/controllers/core/deployer_controller.go#L614-L635

Reply:
credit: @andrew-su

Reply:
Not sure I get what you mean. Are you asking for me to *also* replace `enqueueTrackedResources` that already exists above with the one referenced (and keep `enqueueNamedTrackedConfigMap`) or to come up with a unique function that covers both cases (maybe with an optional name for example)?

Reply:
we can replace both with the linked method.

> maybe with an optional name for example

I don't follow

Reply:
The one I added has `&& a.Meta.GetName() == processorImages` in the condition.

Reply:
ah yes. That's a performance optimization, but is still correct without that check.

I'd like to eventually pull this method into a separate utility so we don't need to repeat it everywhere. If you don't make that change in this PR, we can pick it up then.

Reply:
OK, changed to use that one function.

Notice how I've moved the 
```go
		versionKinds, _, err := r.Scheme.ObjectKinds(t)
		if err != nil {
			panic(err)
		}
```
bit out of the callback though.